Public/Src/Utilities/Utilities/Tracing/BinaryLogReader.cs

```diff
@@ -224,7 +224,12 @@ public EventReadResult ReadEvent()
                         Contract.Assert(LogStream.Position <= (position + header.EventPayloadSize), "Event handler read beyond the event payload");
                     }
 
-                    m_logStreamReader.ReadBytes((int)(m_nextReadPosition.Value - LogStream.Position));
+                    // Instead of computing (m_nextReadPosition.Value - LogStream.Position), just seek forward to m_nextReadPosition.Value (less memory used)

```

Comment:
> // Instead of computing (m_nextReadPosition.Value - LogStream.Position), just seek forward to m_nextReadPosition.Value (less memory used) [](start = 19, length = 138)

change the comment